samples/highcharts/series/legend-color/demo.js

```diff
@@ -0,0 +1,35 @@
+Highcharts.chart('container', {
+
+    title: {
+        text: 'Legend symbol color'
+    },
+
+    series: [
+        {
+            data: [1, 5, 2, 6],
+            legendColor: '#ff0000'
+        },
+        {
```

Comment:
Please pay attention to the consistency of bracket formatting. Check globally for the entire demo.

use

```
[{
}, {
}]
```

instead of
```
[
{
}, {
}
]